engine/src/function.cpp

```diff
@@ -88,6 +88,8 @@ Function::Function(QObject *parent)
     , m_overrideFadeOutSpeed(defaultSpeed())
     , m_overrideDuration(defaultSpeed())
     , m_flashing(false)
+    , m_flashOverrides(false)
+    , m_flashForceLTP(false)
```

Comment:
I wonder if we really want to flash anything other than Scenes.
In other words, are these really needed into the Function class or can they be moved to the Scene class?

Reply:
I'd say let's do this step by step.
Cuestack is a sort of abandoned feature (I haven't even ported it to v5)
For now let's stick to Scene-only flashing.
Thanks

Reply:
Alright. I moved it to the Scene class.